pkg/operator/ceph/controller/mirror_peer.go

```diff
@@ -222,3 +239,47 @@ func expandBootstrapPeerToken(ctx *clusterd.Context, clusterInfo *cephclient.Clu
 	// Return the base64 encoded token
 	return []byte(base64.StdEncoding.EncodeToString(decodedTokenBackToJSON)), nil
 }
+
+func shouldRotateMirrorPeerKeys(c *clusterd.Context, clusterInfo *cephclient.ClusterInfo) (bool, error) {
+	cephObj := &cephv1.CephCluster{}
+	if err := c.Client.Get(clusterInfo.Context, clusterInfo.NamespacedName(), cephObj); err != nil {
+		return false, errors.Wrapf(err, "failed to get cluster %v to get the cephx keys to rotate.", clusterInfo.NamespacedName())
+	}
+	desiredCephVersion := clusterInfo.CephVersion
```

Comment:
In other controllers, `clusterInfo.CephVersion` contains `runningCephVersion`. Let's keep that consistent here also.

For now, I think it would be best to set desired and running to the same thing and add a `// TODO: update when WithCephVersionUpdate is supported` note to the section.

Like this: https://github.com/rook/rook/pull/16029/files#r2205725662